packages/eas-cli/src/graphql/types/credentials/AppleProvisioningProfile.ts

```diff
@@ -15,12 +18,10 @@ export const AppleProvisioningProfileFragmentNode = gql`
     }
```

Comment:
github wont let me comment on the line further up, but you can also refactor `appleTeam` to use the AppleTeamFragment too

```
    appleTeam {
      id
     ...AppleTeamFragment
    }
   ...
   ${print(AppleTeamFragmentNode)}
```

Reply:
Done